app.js

```diff
@@ -0,0 +1,124 @@
+'use-strict';
+//Global Variables
```

Comment:
I see and appreciate the intent. Let this be my reminder to you to move your global variables here!